### PR TITLE
Parse `--blur-*` options as documented in manpage

### PR DESCRIPTION
**Pull request by tryone144**
The manpage documents three options: `--blur-method`, `--blur-size` and `--blur-deviation`

This PR implements parsing of said options in the config file and on the command line.

### CODE REVIEW

File: src/config_libconfig.c [hunk 1]
```diff
@@ -381,9 +381,24 @@ char *parse_config_libconfig(options_t *opt, const char *config_file, bool *shad
...
+		opt->blur_method = method;
+	}
+	// --blur-size
+	config_lookup_int(&cfg, "blur-size", &opt->blur_radius);
+	// --blur-deviation
+	config_lookup_float(&cfg, "blur-deviation", &opt->blur_deviation);
 	// --blur-background
 	if (config_lookup_bool(&cfg, "blur-background", &ival) && ival) {
-		opt->blur_method = BLUR_METHOD_KERNEL;
+		if (!opt->blur_method) {
```
Comment (yshui) [inline src/config_libconfig.c]:
```suggestion
		if (opt->blur_method == BLUR_METHOD_NONE) {
```

Comment (yshui) [reply src/config_libconfig.c]: Can you squash this suggestion into the commit? Thanks

**Comment by yshui** [comment]
Ah, I forgot to implement these.

Thanks a lot for fixing it for me.

**Comment by tryone144** [comment]
Nice. I've implemented the change you requested.
Now I can add my changes to support the dual_kawase method. :)

**Pull request merged**